config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,117 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Do we need to check in this file we don't actually uncomment anything in it?

Reply:
yeah! rails app:upgrade generates you a thing that lets you turn the scary stuff on piece by piece, so you can enjoy new rails features and incrementally turn on some of the potentially breaking improvements. So as time goes on we'll turn on these defaults, then change the `use_framework_defaults` in `application.rb` to `7.0` from `6.1` or whatever it is now.